compose/backend/neurosynth_compose/tests/api/test_crud.py

```
@@ -21,9 +22,15 @@
         ("projects", Project, ProjectSchema),
     ],
 )
-def test_create(session, auth_client, user_data, endpoint, model, schema):
-    user = User.query.filter_by(name="user1").first()
-    examples = model.query.filter_by(user=user).all()
+def test_create(session, auth_client, user_data, db, endpoint, model, schema):
+    user = db.session.execute(
+        select(User).where(User.name == "user1")
+    ).scalar_one_or_none()
+    examples = (
+        db.session.execute(select(model).where(model.user == user)).scalars().all()
+        if user is not None
+        else []
+    )
```

I want this to fail louder, do not have a fallback empty list.